### PR TITLE
Add create playbook dropdown

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR adds create playbook dropdown to the LHS. 
`Invite people` is out of scope
`Start a run` is not implemented, cause when we create runs we use the interactive dialogue and it will not work without the channel. Either we have to implement our own dialogue or fix the current implementation of the interactive dialogue. 

https://user-images.githubusercontent.com/2340127/174095357-41c3b6c3-784e-4914-8303-9d09b11c1d66.mp4

#### Checklist

- [ ] Telemetry updated
- [ ] Gated by experimental feature flag
- [ ] Unit tests updated

### CODE REVIEW

File: webapp/src/components/sidebar/create_playbook_dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+        const importPlaybook = (
+            <>
+                <MenuItem
+                    id='importPlaybook'
+                    show={true}
+                    onClick={() => {
+                        if (fileInputRef && fileInputRef.current) {
+                            fileInputRef.current.click();
+                        }
```
Comment (agarciamontoro) [inline webapp/src/components/sidebar/create_playbook_dropdown.tsx:84]: Optional chaining again?
```suggestion
                    onClick={fileInputRef?.current?.click}
```

File: webapp/src/components/sidebar/create_playbook_dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                    }}
+                    icon={
+                        <StyledMDIIcon
+                            path={mdiImport}
+                            size={1.25}
```
Comment (agarciamontoro) [inline webapp/src/components/sidebar/create_playbook_dropdown.tsx:89]: Something I learned a couple of weeks ago: It turns out you can pass a string here as `12px`, which is way more  meaningful than this. Could we try it here and in the following mdi icons?

File: webapp/src/components/sidebar/create_playbook_dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        color: var(--sidebar-text);
+    }
+
+    min-width: 28px;
+    height: 28px;
+    font-size: 18px;
+    vertical-align: middle;
+
+    &.disabled {
+        background: rgba(255, 255, 255, 0.08);
```
Comment (agarciamontoro) [inline webapp/src/components/sidebar/create_playbook_dropdown.tsx:188]: Should this be themed?

**Review by agarciamontoro — APPROVED** [review]
Neat! :tada: I left some comments below, but none of them blocking.

File: webapp/src/components/sidebar/create_playbook_dropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+interface CreatePlaybookDropdownProps {
+    team_id: string;
+}
+
+const CreatePlaybookDropdown = (props: CreatePlaybookDropdownProps) => {
+    const {formatMessage} = useIntl();
+    const dispatch = useDispatch();
+    const fileInputRef = useRef<HTMLInputElement | null>(null);
+    const teams = useSelector<GlobalState, Team[]>(getMyTeams);
```
[the remark this answers is not in the document]
Comment (calebroseland) [reply webapp/src/components/sidebar/create_playbook_dropdown.tsx:31]: +1, fewer type imports needed

File: webapp/src/components/sidebar/create_playbook_dropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+        const createPlaybook = (
+            <MenuItem
+                id='createPlaybook'
+                show={true}
+                onClick={() => dispatch(displayPlaybookCreateModal({}))}
+                icon={<StyledIcon className='icon-plus'/>}
+                text={formatMessage({defaultMessage: 'Create New Playbook'})}
+            />
+        );
```
Comment (calebroseland) [inline webapp/src/components/sidebar/create_playbook_dropdown.tsx:74]: Shouldn't be behind a permissions gate of some sort? The one in `playbook_list` is behind `useCanCreatePlaybooksOnAnyTeam`.  This/that may require refactoring once we rescope to single-team, but I think we should at least match for now.

**Review by calebroseland — CHANGES_REQUESTED** [review]
Just one UI issue (icon alignment):
CleanShot 2022-06-17 at 16 23 20@2x

Otherwise looks great and will be nice to have this in!

**Review by calebroseland — APPROVED** [review]

**Pull request merged**